ui/src/organizations/components/OrgNavigation.tsx

```diff
@@ -42,11 +43,18 @@ class OrgNavigation extends PureComponent<Props> {
         cloudExclude: true,
         link: `/orgs/${orgID}/members`,
       },
```

Comment:
This is visible always in OSS and never in Cloud. I think this is correct to go to `/members`.